app/bundles/CoreBundle/Form/Type/DynamicContentFilterType.php

```diff
@@ -34,6 +34,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'attr'  => [
                     'class' => 'form-control dynamic-content-token-name',
                 ],
+                'read_only' => true,
```

Comment:
Did you mean to make this read only? JS won't be able to update this dynamically if set as so.